pkgs/by-name/ps/psb_status/package.nix

```diff
@@ -0,0 +1,40 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  bash,
+  iotools,
+  makeWrapper,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "psb_status";
+  version = "0-unstable-2024-10-10";
+
+  src = fetchFromGitHub {
+    owner = "mkopec";
+    repo = "psb_status";
+    rev = "be896832c53d6b0b70cf8a87f7ee46ad33deefc2";
+    hash = "sha256-4anPyjO8y3FgnYWa4bGFxI8Glk9srw/XF552tnixc8I=";
+  };
+
+  dontBuild = true;
+
+  nativeBuildInputs = [ makeWrapper ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    install -m755 psb_status.sh $out/bin/psb_status
+    wrapProgram $out/bin/psb_status \
+      --prefix PATH : ${lib.makeBinPath [ iotools ]}
+  '';
+
+  meta = {
+    description = "A script to check Platform Secure Boot enablement on Zen based AMD CPUs";
+    homepage = "https://github.com/mkopec/psb_status";
+    license = lib.licenses.mit;
+    maintainers = [ ];
```

Comment:
Are you willing to maintain the package? It would be nice to avoid adding one more orphaned package :sweat_smile: